### DISCUSSION TITLE
Please bring back the old feed!

### DISCUSSION THREAD

**Discussion by sureshg** [community/community#66228] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The new feed is horrible for somebody like me who has ~ 5K stars . I lost all my activity feeds of users i am following.  Please bring back the old feed or make the new changes optional. Even filter are not working and all it shows an empty page.

**Comment by flrngel** [comment]
for me who has over 5K stars, new feed is just another disaster

**Comment by shailshouryya** [comment]
I made https://github.com/orgs/community/discussions/66226 and am currently aggregating the current discussions about this there
